bin/generate.dart

```diff
@@ -316,8 +318,20 @@ class DiagramGenerator {
   /// into.
   Directory temporaryDirectory;
 
+  /// The device ID to use when transferring results from Android.
+  String deviceId = '';
+
+  /// The targetPlatform from the `flutter devices` output of the device we're
+  /// targeting.
+  String deviceTargetPlatform = '';
+
   Future<void> generateDiagrams(List<String> categories, List<String> names) async {
     final DateTime startTime = DateTime.now();
+    if (!await _findIdForDeviceName(device)) {
+      stderr.writeln('Unable to find device ID for device $device. Are you sure it is attached?');
```

Comment:
Should `$device` be `deviceId`?

Reply:
No, that's where I convert the device into a deviceId.  I suppose I don't actually have to pass the device, since it's a member var.